EasyPost/Utilities/Http.cs

```diff
@@ -63,10 +73,10 @@ internal static bool ReturnedError(this RestResponse response)
         ///     Return whether the given response has a successful status code.
         /// </summary>
         /// <param name="response">Response to check.</param>
-        /// <returns>True if the response code is in the 200-399 range, false otherwise.</returns>
+        /// <returns>True if the response code is in the 100-299 range, false otherwise.</returns>
         internal static bool ReturnedNoError(this RestResponse response)
         {
-            return StatusCodeBetween(response, 100, 399);
+            return StatusCodeBetween(response, 100, 299);
```

Comment:
Shouldn't the floor be 200 like all the other libs? Discard 1xx as errors too?

Reply:
Resolved